organisaatio-ui/src/components/Sivut/Ryhmat/Ryhmat.tsx

```diff
@@ -4,45 +4,43 @@ import TyypitJaRyhmatKehys from '../TyypitJaRyhmatKehys/TyypitJaRyhmatKehys';
 import { useContext } from 'react';
 import { KoodistoContext, LanguageContext } from '../../../contexts/contexts';
 import Button from '@opetushallitus/virkailija-ui-components/Button';
-import Input from '@opetushallitus/virkailija-ui-components/Input';
-import { Icon } from '@iconify/react';
-import searchIcon from '@iconify/icons-fa-solid/search';
-import Select from '@opetushallitus/virkailija-ui-components/Select';
 import { useEffect } from 'react';
 import Axios from 'axios';
 import { useState } from 'react';
 import { Ryhma } from '../../../types/types';
 import NormaaliTaulukko from '../../Taulukot/NormaaliTaulukko';
 import Spin from '@opetushallitus/virkailija-ui-components/Spin';
+import { Column } from 'react-table';
 
 const urlPrefix = process.env.NODE_ENV === 'development' ? '/api' : '/organisaatio';
 
 const Ryhmat: React.FC = (props) => {
+    const { i18n, language } = useContext(LanguageContext);
     const { ryhmaTyypitKoodisto, kayttoRyhmatKoodisto } = useContext(KoodistoContext);
     const [ryhmat, setRyhmat] = useState<Ryhma[] | undefined>(undefined);
+
     useEffect(() => {
         async function fetch() {
             try {
                 const response = await Axios.get(`${urlPrefix}/organisaatio/v3/ryhmat?aktiivinen=true`);
```

Comment:
it would be very beneficial to abstract all api calls to separate module (and properly add type information for expected responses)